python/pylibraft/pylibraft/common/handle.pxd

```diff
@@ -33,3 +34,8 @@ cdef extern from "raft/core/handle.hpp" namespace "raft" nogil:
                  shared_ptr[cuda_stream_pool] stream_pool) except +
         cuda_stream_view get_stream() except +
         void sync_stream() except +
+
+cdef class Handle:
```

Comment:
It looks like pylibraft already had this in the pyx file. Were these just untyped Python objects before, then? I'm sort of surprised that worked, am I missing something?

Reply:
We just weren't using the `Handle` before, so we didn't explicitly need to build it.